guide/getting-started/visual-language.md

```diff
@@ -203,6 +223,35 @@ Think about Bitcoin’s visual language as a scale that ranges from muted to hig
 
 In digital product design, there are times when a minimally expressive approach is what’s needed. For example, when a user makes a transaction, we want them to focus. At other times, illustrations and stronger visual elements are more appropriate, and this is where we can take advantage of the creative freedom Bitcoin affords.
 
+## The Lightning network
+
+The Lightning network is a layer 2 solution for scaling Bitcoin. It allows people to make almost instant payments with very small fees.
+
+Since the first Lightning implementation in 2018, a distinct visual identity has developed around the Lightning network based on a simple yellow icon of a lightning bolt. Sometimes, Lightning software and services may choose to completely forego common elements of Bitcoin branding (the color orange, the Bitcoin symbol, etc.) and instead use the color yellow, a lightning bolt symbol, and metaphors for electricity.
```

Comment:
Wonder if its worth mentioning the distinct visual branding of the various Lightning implementations. Also the purple circle with the bolt in the middle is what I was familiar with from the early Lightning days, it's a primary image on on Wikipedia also: https://commons.wikimedia.org/wiki/File:Bitcoin_over_Lightning_Network.svg

Reply:
Showing a variety of them instead of the various Lightning wallet logos is my recommendation.

Reply:
c7104e4 I updated this image to include the purple LN icon. I think the purple version actually has been somewhat influential; I feel like purple is the color I see most often associated with LN second to yellow.

@Bosch-0 I don't want to discuss the distinct brandings of different LN implementations -- my hope is that the user never has to worry about things like LND, C-Lightning, or Eclair; and my thinking is that it may be too much detail for a designer who is simply trying to consider the needs of their user.